etc/config/mlir.defaults.properties

```diff
@@ -10,5 +10,12 @@ group.mliropt.compilers=mliropt14
 group.mliropt.isSemVer=true
 group.mliropt.baseName=MLIR opt
 
+group.mlirtranslate.compilers=mlirtranslate14
+group.mlirtranslate.isSemVer=true
+group.mlirtranslate.baseName=MLIR translate
+
 compiler.mliropt14.exe=/usr/bin/mlir-opt-14
 compiler.mliropt14.semver=14.0.0
+
+compiler.mlirtranslate14.exe=/usr/bin/mlir-translate-14
+compiler.mlirtranslate14.semver=14.0.0
```

Comment:
It would be useful to have a local trunk compiler as well, I assume that would be `/usr/bin/mlir-translate`. For anyone who has done `ninja install`.